src/components/ServiceRequestForm.jsx

```diff
@@ -98,26 +98,45 @@ const ServiceRequestForm = (props, errors, touched) => {
 					returnJsonFileLocations("resultFormFieldNames"),
 				);
 
-				setData(result.data);
-				localProps.setFieldValue('Categories', result.data);
+				setCategories(result.data);
 				setPetTypes(resultPetTypes.data);
 				setAnimalBreeds(resultAnimalBreeds.data);
 				setAnimalColors(resultAnimalColors.data);
 				setOtherAnimalTypes(resultAnimalTypes.data);
 
+				const preSelectedTypes = SelectedValue(result.data);
+
+				let requestCategory ='';
+				let requestSubCategory = ''
+
+				const selectedType = () =>{
+					 requestCategory = (preSelectedTypes) ? preSelectedTypes.nameCategory : requestType ;
+					if(requestCategory){		
+						addSelectedValueOptions(result.data, requestCategory.toLowerCase());
+					}
+					return requestCategory;
+				}
+
+				const selectedSubType = () =>{
+					 requestSubCategory = (preSelectedTypes) ? preSelectedTypes.nameSubCategory : subRequestType ;
+					if(requestSubCategory){
+						addSelectedSubValueOptions(result.data, requestSubCategory.toLowerCase())
+					}
+					return requestSubCategory;
+				}
+
 				localProps.setFieldValue('Tabs', resultFormFieldNames.data.Tabs);
 				localProps.setFieldValue('RequestPage', resultFormFieldNames.data.RequestPage);
 				localProps.setFieldValue('MapPage', resultFormFieldNames.data.MapPage);
 				localProps.setFieldValue('AdditionalInfoPage', resultFormFieldNames.data.AdditionalInfoPage);
 				localProps.setFieldValue('SignInPage', resultFormFieldNames.data.SignInPage);
 				localProps.setFieldValue('SignUpPage', resultFormFieldNames.data.SignUpPage);
 				localProps.setFieldValue('ResetPasswordPage', resultFormFieldNames.data.ResetPasswordPage);
-
+				localProps.setFieldValue('requestType', selectedType());
```

Comment:
**not for this pr** but we should probably consdider coming up wtih some function that you can pass either an object or an array to set field values, something like

```js
const setFieldValues = (props, fields) => {
  fields.forEach(field => {
    const { name, value } = field;
    props.setFieldValue(name, value);
  });
}

Reply:
Can we put in a backlog item for this? I think this would be helpful.

Reply:
yes, I'll hide it near the bottom somewhere

Reply:
tracked here https://github.com/baltimorecounty/react-baltcogo/issues/103